src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskExtensions.cs

```diff
@@ -49,10 +49,13 @@ public static Task<TResult> Unwrap<TResult>(this Task<Task<TResult>> task)
                 Task.FromCanceled<TResult>(new CancellationToken(true));
         }
 
+        // TODO: Remove the below three methods once corefx has consumed a build with them in their new TaskAsyncEnumerableExtensions location.
```

Comment:
Do we have an issue to track this work?

Reply:
We don't.  My plan is to remove this as soon as it's consumed into corefx.  I'll open an issue if that doesn't happen shortly.